src/controller/abstractMaze2DGenerator.java

```diff
@@ -0,0 +1,24 @@
+package controller;
+
+public abstract class abstractMaze2DGenerator implements maze2dGenerator {
+
+    // Measuring generating algorithm execution time, returning millis as String
+    public String measureAlgorithmTime(int mazeSize) throws mazeSizeException {
+        String retStr;
+
+        if (mazeSize < 3) {
+            throw new mazeSizeException("maze size has to be bigger then 2");
```

Comment:
Why the magic number of 3?

Reply:
I assume that the minimum size of a valid maze is 3x3. I do not want smaller mazes

Reply:
ok